.golangci.yml

```diff
@@ -17,7 +17,14 @@ issues:
   exclude-rules:
    # Exclude duplicate code and function length and complexity checking in test
    # files (due to common repeats and long functions in test code)
-   - path: _test\.go #TODO: also exclude all go-generated files by giving them common names? *_gen.go?
+   - path: _test\.go
+     linters:
+       - gocyclo
+       - dupl
+       #TODO - gocognit
+       - funlen
+       - lll
+   - path: _gen\.go
```

Comment:
I didn't see a way to repeat paths for the same list of linters to exclude, hence the repetition.

We could probably hack around it by using [YAML anchors](https://confluence.atlassian.com/bitbucket/yaml-anchors-960154027.html), but I'd rather not. :)

Reply:
I think path is a regex so you can probably write a small regex like `_(test|gen)\.go` and it will work

Reply:
You're right, missed that detail. :) Will fix.